sdk/paloaltonetworks/azure-mgmt-paloaltonetworksngfw/tests/test_cli_mgmt_paloaltonetworksngfw.py

```diff
@@ -0,0 +1,38 @@
+# coding: utf-8
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import unittest
+import pytest
+
+import azure.mgmt.paloaltonetworksngfw
+from devtools_testutils import AzureMgmtRecordedTestCase, RandomNameResourceGroupPreparer, recorded_by_proxy
+
+AZURE_LOCATION = 'eastus'
+
+class TestMgmtPaloaltonetworksngfw(AzureMgmtRecordedTestCase):
+
+    def setup_method(self, method):
+        self.mgmt_client = self.create_mgmt_client(
+            azure.mgmt.paloaltonetworksngfw.PaloAltoNetworksNgfwMgmtClient
+        )
+    
+    @RandomNameResourceGroupPreparer(location=AZURE_LOCATION)
+    @recorded_by_proxy
+    def test_paloaltonetworksngfw(self, resource_group):
+        local_rulestack_name = self.get_resource_name("localrulestack")
```

Comment:
What is the usage of `get_resource_name`

Reply:
Inherit from parent class, to get random resource name